scripts/update_images.py

```diff
@@ -0,0 +1,178 @@
+    return new_file_path
+
+def get_helm_operator_image(related_images: list)->dict:
+    """Get the settings for Helm Operator in the list of related images
+    
+    Args:
+        related_images (str): list of related images
+    
+    Returns:
+        dict: Helm Operator setting in the list of related images
+    """
+    operator_image = None
+    for image in related_images:
+        if image["name"] == "sumologic-kubernetes-collection-helm-operator":
+            operator_image = image
+    return operator_image
+
+def update_envs(envs: list, new_image_envs) -> list:
+    """Updates environment variables, add new environment variables with information about components images
+    and preserves not image related environment variables
+
+    Args:
+        envs (list): list of environment variables
+        new_image_envs (list): list of new environment variables with information about components images
+        create_new_file (bool): determines whether new yaml should be created or the exiting file should be overwritten 
+
+    Returns
+        list: update list of list of environment variables
+    """
+    not_image_envs = []    
+    for env in envs:
+        if ENV_PREFIX not in env["name"]:
+            not_image_envs.append(env)
+    return not_image_envs + new_image_envs
+
+def update_cluster_service_version(file_path: str, new_related_images: list, new_image_envs: list, create_new_file):
+    """ Updates components images in bundle/manifests/operator.clusterserviceversion.yaml
+
+    Args:
+        file_path (str): absolute path to bundle/manifests/operator.clusterserviceversion.yaml
+        new_related_images (list): list of new related images
+        new_image_envs (list): list of new environment variables with information about components images
+        create_new_file (bool): determines whether new yaml should be created or the exiting file should be overwritten 
+    """
+    with open(file_path) as cr:
+        cluster_service_version = yaml.safe_load(cr)
+        related_images = cluster_service_version["spec"]["relatedImages"]
+
+        helm_operator_image = get_helm_operator_image(related_images)
+        new_related_images.insert(0, helm_operator_image)
+        cluster_service_version["spec"]["relatedImages"] = new_related_images
+
+        # todo: search for the helm operator container
+        for i in range(len(cluster_service_version["spec"]["install"]["spec"]["deployments"][0]["spec"]["template"]["spec"]["containers"])):
+            name = cluster_service_version["spec"]["install"]["spec"]["deployments"][0]["spec"]["template"]["spec"]["containers"][i]["name"]
+            if name == "operator":
+                envs = cluster_service_version["spec"]["install"]["spec"]["deployments"][0]["spec"]["template"]["spec"]["containers"][i]["env"]
+                cluster_service_version["spec"]["install"]["spec"]["deployments"][0]["spec"]["template"]["spec"]["containers"][i]["env"] = update_envs(envs, new_image_envs)
```

Comment:
I would assign `cluster_service_version["spec"]["install"]["spec"]["deployments"][0]["spec"]["template"]["spec"]["containers"]` to variable to increase readability